app/concerns/twitter_concern.rb

```diff
@@ -36,7 +36,7 @@ def twitter_oauth_token_secret
   end
 
   def twitter
-    Twitter::REST::Client.new do |config|
+    @twitter ||= Twitter::REST::Client.new do |config|
```

Comment:
I *think* this should be fine-- we'd be making a new one for any instance of a new Agent anyway.